craigslist-apartments.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+let axios = require('axios');
+let cheerio = require('cheerio');
+let filters = require('./filters');
+
+const Filter = new filters();
+let mountedURL = mountURL(Filter); 
+
+function mountURL(filter) {
+ let url = `https://vancouver.craigslist.org/search/apa?availabilityMode=0&bundleDuplicates=1&hasPic=1`;
+ if(filter.getMaxPrice()) {
+   url = url + `&max_price=${filter.getMaxPrice()}`;
+ }
+ if(filter.getMinPrice()) {
+   url = url + `&min_price=${filter.getMinPrice()}`; 
+ }
+ if(filter.getKmFromPostalCode()) {
+   url = url + `&search_distance=${filter.getKmFromPostalCode()}`; 
+ }
+ if(filter.getPostalCode()) {
+   url = url + `&postal=${filter.getPostalCode()}`; 
+ }
+ if(filter.getMinSqft()) {
+   url = url + `&minSqft=${filter.getMinSqft()}`; 
+ }
+ return url;
+}
+
+function getListOfRentals(totalLastSearch, listOfRentals) {
+  let url = mountedURL + `&s=${totalLastSearch}`;
```

Comment:
instead of using this `mountedURL`, you should call the method here passing the filters and the `totalLastSearch` (I think `startIndex` is a better name)

Reply:
I have changed the name 'totalLastSearch' to startIndex, but I am not passing the 'totalLastSearch' to the function mountURL because part of the URL doesn't need to be created more the once and I created other function that calls the function mountURL.